src/pages/signin/LoginForm.js

```diff
@@ -75,10 +90,11 @@ class LoginForm extends React.Component {
                     <Text style={[styles.formLabel]}>{this.props.translate('loginForm.enterYourPhoneOrEmail')}</Text>
                     <TextInput
                         style={[styles.textInput]}
-                        value={this.state.login}
+                        ref={this.login}
+                        value={this.login}
```

Comment:
Yes, this should be an uncontrolled component. 
Which means it will not have a `value` prop defined. Please see examples here:

https://reactjs.org/docs/uncontrolled-components.html

>But, state will be lost if I don't use this here (When Parent comp. changes)

To clarify, why is the parent component changing? The input must have a `defaultValue` set in order for the value to be preserved if the component remounting is causing the value to be lost.

Reply:
It changes when the width moves around `800px` in `SignInPageLayout`

https://github.com/Expensify/App/blob/8c06c02f5917b2d9cd49aaa2442029691f0794e0/src/pages/signin/SignInPage.js#L87-L96

https://github.com/Expensify/App/blob/8c06c02f5917b2d9cd49aaa2442029691f0794e0/src/pages/signin/SignInPageLayout/index.js#L10-L16


> Which means it will not have a value prop defined

But, now that you've said it..I think I have the solution now, we can just use `defaultValue` prop to prevent the `loss of state` as we can set `initial value` through this.